### PR TITLE
get the city name and id

### PR DESCRIPTION
**Pull request by alMutazBeAllah**
- return all of cities name and id as a JSON object into the client.
 - populate a checklist cities name

### CODE REVIEW

File: public/main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    return response.json();
+})
+.then((cities)=>{
+    var list = document.getElementById("select_city");
+    cities.forEach(element => {
+        var op = document.createElement("option");
+        op.innerText= element.city_name;
+        op.value = element.id;
+        list.appendChild(op);
+    });
```
Comment (ghassanmas) [inline public/main.js]: You could use `let` instead of `var` since you are using fetch which is ES6

File: public/main.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+fetch('/getcities')
+.then(response => {
+    return response.json();
+})
+.then((cities)=>{
+    var list = document.getElementById("select_city");
```
Comment (ghassanmas) [inline public/main.js]: You could use `let` instead of `var` since you are using fetch which is ES6

File: src/handlers.js [hunk 3]
```diff
@@ -71,5 +70,17 @@ const addPostHandler = (request,response) =>{
     });
   });
 }
+// -- GET CITIES HANDLER -------------------
+const getCitiesHandler = (request, response) => {
+    getCities((error, result) => {
+        if(error){
+            console.log(error)
+            return errorNotFound(error, response)
+        }
```
Comment (ghassanmas) [inline src/handlers.js]: Should response with 500 server side Error

File: public/main.js (hunk 2 above)
Comment (alMutazBeAllah) [reply public/main.js]: we will do it

**Pull request merged**